oneBarangay/static/assets/gulpfile.js

```diff
@@ -16,92 +16,92 @@
 
 */
 
-const autoprefixer = require("gulp-autoprefixer");
-const browserSync = require("browser-sync").create();
-const cleanCss = require("gulp-clean-css");
-const del = require("del");
-const htmlmin = require("gulp-htmlmin");
-const cssbeautify = require("gulp-cssbeautify");
-const gulp = require("gulp");
-const npmDist = require("gulp-npm-dist");
-const sass = require("gulp-sass")(require("node-sass"));
-const wait = require("gulp-wait");
-const sourcemaps = require("gulp-sourcemaps");
-const fileinclude = require("gulp-file-include");
-const rename = require("gulp-rename");
+const autoprefixer = require('gulp-autoprefixer')
+const browserSync = require('browser-sync').create()
+const cleanCss = require('gulp-clean-css')
+const del = require('del')
+const htmlmin = require('gulp-htmlmin')
+const cssbeautify = require('gulp-cssbeautify')
+const gulp = require('gulp')
+const npmDist = require('gulp-npm-dist')
+const sass = require('gulp-sass')(require('node-sass'))
+const wait = require('gulp-wait')
+const sourcemaps = require('gulp-sourcemaps')
+const fileinclude = require('gulp-file-include')
+const rename = require('gulp-rename')
 
 // Define paths
 
 const paths = {
   dist: {
-    base: "./dist/",
-    css: "./dist/css",
-    html: "./dist/pages",
-    assets: "./dist/assets",
-    img: "./dist/assets/img",
-    vendor: "./dist/vendor",
+    base: './dist/',
+    css: './dist/css',
+    html: './dist/pages',
+    assets: './dist/assets',
+    img: './dist/assets/img',
+    vendor: './dist/vendor'
   },
   dev: {
-    base: "./html&css/",
-    css: "./html&css/css",
-    html: "./html&css/pages",
-    assets: "./html&css/assets",
-    img: "./html&css/assets/img",
-    vendor: "./html&css/vendor",
+    base: './html&css/',
+    css: './html&css/css',
+    html: './html&css/pages',
+    assets: './html&css/assets',
+    img: './html&css/assets/img',
+    vendor: './html&css/vendor'
   },
   base: {
-    base: "./",
-    node: "./node_modules",
+    base: './',
+    node: './node_modules'
   },
   src: {
-    base: "./",
-    css: "./css",
-    html: "./src/pages/**/*.html",
-    assets: "./src/assets/**/*.*",
-    partials: "./src/partials/**/*.html",
-    scss: "./scss",
-    node_modules: "./node_modules/",
-    vendor: "./vendor",
+    base: './',
+    css: './css',
+    html: './src/pages/**/*.html',
+    assets: './src/assets/**/*.*',
+    partials: './src/partials/**/*.html',
+    scss: './scss',
+    node_modules: './node_modules/',
+    vendor: './vendor'
   },
   temp: {
-    base: "./.temp/",
-    css: "./.temp/css",
-    html: "./.temp/pages",
-    assets: "./.temp/assets",
-    vendor: "./.temp/vendor",
-  },
-};
+    base: './.temp/',
+    css: './.temp/css',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PrynsTag/oneBarangay/pullRequest?prid=7999497)
```suggestion
        css: "./.temp/css",
```